envs/monkey_zoo/blackbox/island_client/monkey_island_requests.py

```diff
@@ -61,17 +65,21 @@ def try_get_jwt_from_server(self):
     def get_jwt_from_server(self):
         resp = requests.post(  # noqa: DUO123
             self.addr + "api/auth",
-            json={"username": NO_AUTH_CREDS, "password": NO_AUTH_CREDS},
+            json={"username": ISLAND_USERNAME, "password": ISLAND_PASSWORD},
             verify=False,
         )
         if resp.status_code == 401:
             raise AuthenticationFailedError
         return resp.json()["access_token"]
 
-    def try_set_island_to_no_password(self):
-        requests.patch(  # noqa: DUO123
-            self.addr + "api/environment", json={"server_config": "standard"}, verify=False
+    def try_set_island_to_credentials(self):
```

Comment:
This function name doesn't make a lot of sense. How about `try_log_in_to_island()` or `try_island_login()`?